onnxruntime/python/tools/transformers/models/phi2/inference_example.py

```diff
@@ -136,13 +224,38 @@ def generate(self, prompt, max_length):
         current_length = sequence_length
         has_eos = torch.zeros(batch_size, device=self.device, dtype=torch.bool)
 
+        if benchmark:
+            import time
+
+            latency = []
+
+        prompt_run = True
         while current_length < max_length:
             io_binding = self.apply_io_binding(self.sess, inputs, outputs)
 
+            if benchmark:
+                start = time.time()
+
             io_binding.synchronize_inputs()
-            self.sess.run_with_iobinding(io_binding)
+            if prompt_run:
```

Comment:
Add one shared library C++ test as well ? Maybe it can be trying to capture one "session level" graph and one "run level graph" and replaying them both with different inputs.

Reply:
would this be enough? https://github.com/microsoft/onnxruntime/pull/19636/files/29fd4ff6eccec6d4e55d02b944860807d4f9f1cc#diff-78b6f9eda1d96f70a3700f385e8339b70841129e827482bc4d3537e1e800d464

Reply:
Maybe add one where both are run together in the same session (non-graph id capture and graph id based capture happening with same session instance) ?

Reply:
sure